connector/src/main/java/org/geysermc/connector/utils/Toolbox.java

```diff
@@ -52,6 +52,8 @@
 
     public static final Int2ObjectMap<ItemEntry> ITEM_ENTRIES = new Int2ObjectOpenHashMap<>();
 
+    public static final HashMap<String, HashMap<String, String>> LOCALE_MAPPINGS = new HashMap<>();
```

Comment:
Don't use `HashMap` unless you are initializing the map. Should just be `Map<String, Map<String, String>> LOCALE_MAPPINGS = new HashMap<>();`